third_party/crashpad/crashpad/client/crashpad_info.cc

```diff
@@ -97,13 +97,15 @@ extern "C" int* CRASHPAD_NOTE_REFERENCE;
 CrashpadInfo* CrashpadInfo::GetCrashpadInfo() {
 #if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_ANDROID) || \
     BUILDFLAG(IS_FUCHSIA)
+#if !BUILDFLAG(ENABLE_COBALT_HERMETIC_HACKS)
```

Comment:
I've been adding `!BUILDFLAG(FOO)` as `BUILDFLAG(IS_LINUX) && !BUILDFLAG(FOO)` when there's an existing macro guard, but I don't really have a preference to doing it one way or another or a simple condition like this that we'll be addressing later anyways